test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -73,6 +73,40 @@ public void PackCommand_PackNewDefaultProject_NupkgExists()
             }
         }
 
+        [PlatformFact(Platform.Windows)]
+        public void PackCommand_PackToolUsingAlias_DoesNotWarn()
```

Comment:
naming: You are saying `DoesNotWarn`, but the test case is more specific than that. 

I'd recommend changing the name to reflect that.